docs/learning/04 Testing.ipynb

```diff
@@ -383,9 +383,9 @@
    "source": [
     "## Test Driven Development\n",
     "\n",
-    "I won't talk about TDD much here but it's likely a term you will hear at some point. It essentially referes to the practice of writing tests as part of your process of writing code. Rather than writing all your code and then writing tests for them. You could instead write some or all of your tests upfront and then write code that passes them. \n",
+    "I won't talk about TDD much here, but it's likely a term you will hear at some point. It essentially refers to the practice of writing tests as part of your process of writing code. Rather than writing all your code and then writing tests for them. You could instead write some or all of your tests upfront and then write code that passes them. \n",
```

Comment:
```suggestion
    "I won't talk much about Test Driven Development, or TDD, here, but it's likely a term you will hear at some point. It essentially refers to the practice of writing tests as part of your process of writing code. Rather than writing all your code and then writing tests for them. You could instead write some or all of your tests upfront, describing the expected bahviour of code that doesn't yet exist, and then write the necessary code so that your tests pass. \n",
```